src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final Ball winner = new Ball();

```

Comment:
Having 'winner' as a class-level variable may lead to unwanted behavior since the same Ball instance is used every time 'getRandomBall()' is called. Instead, you should create a new Ball instance within the 'getRandomBall()' method to ensure that each call generates a unique Ball object.